### PR TITLE
open source page header done

### PR DESCRIPTION
**Pull request by hrutik7**
## Fixes Issue

This PR fixes the following issues:

closes #issue-number

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes

### CODE REVIEW

**Comment by Chandraprakash-Darji** [comment]
@hrutik7 I would recommend when ever you create PR, just share the supporting screenshot.

**Comment by hrutik7** [comment]
ok

**Comment by hrutik7** [comment]
image

File: src/components/PageComp/OpenSourcePage/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import ArrowLink from '@/components/links/ArrowLink';
+import ButtonLink from '@/components/links/ButtonLink';
+import HeroImg from '@/assets/opensource/collab.svg';
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/OpenSourcePage/Header.jsx:3]: Put assets in public folder, we are shifting to use public folder

File: src/components/PageComp/OpenSourcePage/About.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import AboutImage from 'assets/home/aboutus.png';
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/OpenSourcePage/About.jsx:1]: try to find some image about open source
image

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/components/PageComp/OpenSourcePage/Header.jsx (hunk 1 above)
Comment (hrutik7) [reply src/components/PageComp/OpenSourcePage/Header.jsx:3]: ok understood

File: src/components/PageComp/OpenSourcePage/About.jsx (hunk 2 above)
Comment (hrutik7) [reply src/components/PageComp/OpenSourcePage/About.jsx]: image

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by Chandraprakash-Darji** [comment]
Solve the conflict

File: src/components/PageComp/OpenSourcePage/About.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import ColumnSection from '@/components/layout/ColumnSection';
+import ListItem from '@/components/Listitem';
+
+const About = ({ background = 'static' }) => {
```
[the remark this answers is not in the document]
Comment (hrutik7) [reply src/components/PageComp/OpenSourcePage/About.jsx:4]: ok @Chandraprakash-Darji understood

File: src/assets/opensource/collab.svg
Comment (Chandraprakash-Darji) [inline src/assets/opensource/collab.svg:1]: Remove this asset file

File: src/components/PageComp/OpenSourcePage/Header.jsx (hunk 1 above)
Comment (Chandraprakash-Darji) [inline src/components/PageComp/OpenSourcePage/Header.jsx:3]: This import line is not required I guess

File: src/components/PageComp/OpenSourcePage/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+          <h1 className='heading heading-md highlight text-left md:text-center lg:text-left'>
+          Open source, collaboration for innovation
+          </h1>
+          <p className='h3 my-9'>
+            The Power of Open Source: Unleashing the Collective Genius of Global
+            Communities
+          </p>
+        </div>
+        <div className='relative mt-0 max-w-lg  py-8 lg:mt-16 lg:py-16 lg:px-8 '>
+          <img src={`/assets/images/OpenSource/collab.svg`} className='w-full' />
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/OpenSourcePage/Header.jsx:18]: Remove the curly bracket, because it will produce the prettier and eslint error.

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/components/PageComp/OpenSourcePage/Header.jsx (hunk 4 above)
Comment (hrutik7) [reply src/components/PageComp/OpenSourcePage/Header.jsx:18]: ok

**Review by Chandraprakash-Darji — APPROVED** [review]

**Pull request merged**